dbms/src/Storages/Transaction/RegionTable.h

```diff
@@ -211,7 +216,10 @@ class RegionTable : private boost::noncopyable
     InternalRegion & insertRegion(Table & table, const Region & region);
     InternalRegion & doGetInternalRegion(KeyspaceTableID ks_tb_id, RegionID region_id);
 
-    RegionDataReadInfoList flushRegion(const RegionPtrWithBlock & region, bool try_persist) const;
+    // Try write the committed kvs into cache of columnar DeltaMergeStore.
+    // Flush the cache if try_persist is set to true.
+    // The original name for this method is flushRegion.
```

Comment:
```suggestion
```

Reply:
Why?

Reply:
I think we don't need to keep this message... it doesn't help